docs/UserGuide.md

```diff
@@ -584,6 +584,7 @@ However, parameters with no prefix (eg: `INDEX`) must appear as the first parame
 7. **The error message** for the `EMAIL` field in a company's details does not specify the exact reason for format violations (e.g., each segment of the `domain` between periods must be at least 2 characters long). For detailed formatting rules and examples, please refer to the notes in [features](#features).
 8. `InternBuddy` **currently accepts invalid phone numbers**, such as those exceeding 15 digits. Users who enter an incorrect phone number by mistake can correct it using the `edit` command.
 9. `InternBuddy` **currently has no limit on the tag length.** However, with the default window size, the tag may overrun the screen if it exceeds 62 characters, causing the favourites star to no longer remain visible. If a tag length longer than 62 characters is required, adjustments to the window size can be made accordingly.
+10. `InternBuddy` **currently accepts multiple applications with the same name and descriptions.** In the real world, one may apply for the same role in a company multiple times, hence this is not considered to be a bug to be rectified.
```

Comment:
I think that this would potentially work better as a QnA response. `Why can I add multiple applications of the same name to a company?`